common/src/main/java/com/microsoft/identity/common/exception/ArgumentException.java

```diff
@@ -60,4 +62,9 @@ public String getArgumentName() {
         return mArgumentName;
     }
 
+    @Override
+    public String getExceptionName(){
+        return sName;
```

Comment:
return ArgumentException.class.getName();?

Here and everywhere.

Reply:
I'm using `sName` to compare , so that in future if the value of `sName` is changed , the logic would work as expected